sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/ChangeFeedEncryptionProcessorBuilder.java

```diff
@@ -21,7 +21,7 @@
  * Helper class to build a encryption supported {@link ChangeFeedProcessor} instance.
  *
  */
-public class ChangeFeedEncryptionProcessorBuilder {
+public final class ChangeFeedEncryptionProcessorBuilder {
```

Comment:
IMO these should only be final where they are final in the Core SDK for consistency reasons?

Reply:
Seems weird that ChangeFeedEncryptionProcessorBuilder is final but ChangeFeedProcessorBuilder is not?

Reply:
Yea we discussed that, we should do final when we know we don't want cx to extend it. In cosmos core there is inconsistency  and no actual reason why we didn't do it. In future if use case comes we can open it again ,as it wont be breaking change